apis/openstack-swift/src/test/java/org/jclouds/openstack/swift/v1/TempAuthMockTest.java

```diff
@@ -71,8 +83,10 @@ public void testTempAuthRequest() throws Exception {
       assertEquals(listContainers.getHeader("X-Auth-Token"), "token");
    }
 
-   private SwiftApi api(String authUrl) throws IOException {
-      Properties overrides = new Properties();
+   private SwiftApi api(String authUrl, Properties overrides) throws IOException {
+      if (overrides == null){
+         overrides = new Properties();
+      }
```

Comment:
Instead of passing null, remove this check and call this method with a new Properties object.